CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+
+## [3.5.8] - 2023-03-16
+### Added
+- Create new layout for My Returns List and Request Returns Available List (change from table to Grid Layout with cards)
+
+## [3.5.7] - 2023-03-15
+### Fixed
+- Change structure of solicitation details and new request (from table to card) in mobile devices.
+
```

Comment:
We should leave the `### Added` and `### Fixed` sections under `## [Unreleased]` since actions take care of this to keep the CHANGELOG up to date.

Reply:
Didn't catch what you mean.
Like this? https://github.com/vtex-apps/return-app/pull/208/commits/1e5c2c763a6667b3f3144f8f0b74361ea8372755

Reply:
@lucasayb yes, exactly.
Sorry that wasn't clear enough.

But we should also remove both `## [3.5.8] - 2023-03-16` and the `## [3.5.7] - 2023-03-15`. Github actions will create these and put the relevant information under them.